DuckDuckGo.Fluent.Plugin/HttpCalls.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading.Tasks;
+using static DuckDuckGo.Fluent.Plugin.JsonResult;
+
+namespace DuckDuckGo.Fluent.Plugin
+{
+    public static class HttpCalls
+    {
+        private const string UserAgentString = "Mozilla/5.0 (Windows NT 10.0; Win64; x64)";
+        private static readonly JsonSerializerOptions SerializerOptions = new() { PropertyNameCaseInsensitive = true };
+
+        public static async Task<DuckDuckGoApiResult> GetApiResult(string url)
```

Comment:
Http calls through `HttpClient` support `CancellationToken` to cancel the call before it ends. 
FS pass you a `CancellationToken` in the search method that will be invoked when user closes FS or research something. You should pass it here so you can use it in the `HttpClient` methods

Reply:
Yes, passing `CancellationToken` now. Since FS doesn't provide `CancellationToken` in all of the methods, I'm just passing `default` wherever possible.